maintainer/check_codestyle.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+echo "check_codestyle.sh CODESTYLE=$1 TRAVIS_PULL_REQUEST=$2 TRAVIS_COMMIT=$3 TRAVIS_COMMIT_RANGE=$4 TRAVIS_REPO_SLUG=$5"
+
+CODESTYLE=$1
+TRAVIS_PULL_REQUEST=$2
+TRAVIS_COMMIT=$3
+TRAVIS_COMMIT_RANGE=$4
+[[ $5 ]] && TRAVIS_REPO_SLUG=$5 || TRAVIS_REPO_SLUG="espressopp/espressopp"
+
+INCORRECT_FILES=""
+
+function clangformat() {
+    for fchanged in $1; do
+        if [[ "${fchanged}" =~ .*\.[ch]+pp ]]; then
+            clang-format -style=file ${fchanged} | diff -q - ${fchanged} &> /dev/null || INCORRECT_FILES="${INCORRECT_FILES} ${fchanged}"
+        fi
+    done
+}
+
+#function pep8format() {
+#}
+
+if [ "X$CODESTYLE" = "XON" ]; then
+    if [ "X$TRAVIS_PULL_REQUEST" != "Xfalse" ]; then
+        # Gets list of files from pull request.
+        FILES="`curl -s "https://api.github.com/repos/${TRAVIS_REPO_SLUG}/pulls/${TRAVIS_PULL_REQUEST}/files" | grep '"filename":' | grep '"filename":' | sed -e 's/.*"filename": "\(.*\)",/\1/g' | xargs`"
```

Comment:
Couldn't we do that with git instead, something along the line of:
```
git diff --stat ${TRAVIS_BRANCH}..HEAD
```

Reply:
Changed